SETUP.md

```diff
@@ -81,23 +85,57 @@ git push --set-upstream origin output
 
 Manubot integrates with cloud services to perform continuous integration (CI).
 For Manubot that means automatically building and deploying your manuscript.
-Manubot supports the following services:
+Manubot supports the following CI services:
+
+| Service | Default | Artifacts | Deployment | Config | Private Repos |
+|---------|---------|-----------|---------|--------|---------------|
+| [GitHub Actions](https://github.com/features/actions) | ✔️ | ✔️ | ✔️ | [`manubot.yaml`](.github/workflows/manubot.yaml) | 2,000 minutes per month |
+| [Travis CI](https://travis-ci.com) | ❌ | ❌ | ✔️ needs setup | [`.travis.yml`](.travis.yml) | 100 build trial |
+| [AppVeyor](https://www.appveyor.com/) | ❌ | ✔️ with PR comments | ❌ | [`.appveyor.yml`](.appveyor.yml) | 14 day trial |
+
+Notes on table fields:
+
+- **Default**: Whether the following uncollapsed setup instructions enable the service by default.
+- **Artifacts**: Manuscript outputs that are saved alongside the CI build logs.
+  Both GitHub Actions and AppVeyor upload the rendered manuscript as an artifact for pull request builds.
+  However, only AppVeyor comments on pull requests with a download link to the artifacts ([example](https://github.com/manubot/rootstock/pull/262#issuecomment-519944731)).
+- **Deployment**: Whether the CI service can write outputs back to the GitHub repository (to the `output` and `gh-pages` branches).
+  Deployment provides GitHub Pages with the latest manuscript version to serve to the manuscript's URL.
+  GitHub Actions will deploy by default without any additional setup.
+  Travis CI will only deploy if an SSH Private Key is provided.
+  To avoid deploying a manuscript multiple times, disable GitHub Actions before providing an SSH Private Key to Travis.
+- **Config**: File configuring what operations CI will perform.
+- **Private Repos**: Quota for private repos.
+  Only GitHub Actions supports cost-free builds of private repositories beyond a trial period.
+  All services are cost-free for public repos.
+
+### GitHub Actions
+
+GitHub Actions is the recommended default CI service because it requires no additional setup.
```

Comment:
It looks like GitHub Actions is enabled by default, right?  The owner has to explicitly disable it in the repository settings.  If that's accurate, these instructions make sense.

Reply:
In the most common case, adding the workflow file should be enough to enable actions. However, it is possible that actions could be disabled at the:

1. repo level
2. [organization level](https://github.blog/2019-10-03-github-actions-adds-policy-controls-for-organization-and-repository-owners/)

Not sure how common this will be.